### PR TITLE
editor: only spawn EDITOR from host if unpriv user NS are enabled

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: snapcraft/wrappers/remote-viewer [hunk 1]
```diff
@@ -45,7 +45,7 @@ if [ "${USERNS}" = "0" ]; then
     echo ""
     echo "Your system has unprivileged user namespaces disabled, as a result, "
     echo "you will need to manually run one a SPICE client with the URI above."
-    sleep infinity
+    exec sleep infinity
```
Comment (tomponline) [inline snapcraft/wrappers/remote-viewer:48]: Why is this change needed?

Comment (simondeziel) [reply snapcraft/wrappers/remote-viewer:48]: That's not needed but it caught my eye. The `exec` ensures no `sh` parent sticks around for no purpose because obviously, the `sleep` is terminal (nothing will execute after) and will never return anyway.

**Review by monstermunchkin — APPROVED** [review]
